templates/CRM/Contribute/Form/AdditionalPayment.tpl

```diff
@@ -97,6 +97,20 @@
       <td>
         <span id='totalAmount'>{$form.currency.html|crmAddClass:eight}&nbsp;{$form.total_amount.html|crmAddClass:eight}</span>&nbsp; <span class="status">{if $paymentType EQ 'refund'}{ts}Refund Due{/ts}{else}{ts}Balance Owed{/ts}{/if}:&nbsp;{$paymentAmt|crmMoney}</span>
       </td>
+      {if $email and $outBound_option != 2}
```

Comment:
this seems fine - didn't Pradeep also rip out a tonne of js that is not required now